pointerevents/pointerevent_click_during_capture-manual.html

```diff
@@ -0,0 +1,90 @@
+        <title>click event target during capture</title>
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" type="text/css" href="pointerevent_styles.css">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="pointerevent_support.js"></script>
+        <style>
+        .box {
+          margin: 10px;
+        }
+        #grey {
+          background: grey;
+        }
+        #blue {
+          background: blue;
+        }
+        #green {
+          background: green;
+        }
+        </style>
+        <script type="text/javascript">
+            PhaseEnum = {
+              Phase1:  "phase1",
+              Phase2:  "phase2",
+            }
+            var phase;;
+            var receivedEvents;
+
+            function resetTestState() {
+              phase = PhaseEnum.Phase1;
+              receivedEvents = [];
+            }
+        
+            function run() {
+                var test_pointerEvent = setup_pointerevent_test("click target during capture", ['mouse']);
+                var grey = document.getElementById('grey');
+                var blue = document.getElementById('blue');
+                var green = document.getElementById('green');
+                
+                ['gotpointercapture', 'lostpointercapture', 'pointerdown', 'pointerup', 'click'].forEach(function(eventName) {
+                  [grey, blue, green].forEach(function(target) {
+                    target.addEventListener(eventName, function(event) {
+                      if (event.eventPhase == event.AT_TARGET) {
+                        receivedEvents.push(event.type + '@' + target.id);
+                        if (target == blue && event.type == 'lostpointercapture') {
+                          if (phase == PhaseEnum.Phase1) {
+                            test_pointerEvent.step(function() {
+                              assert_equals(receivedEvents.join(','), 'pointerdown@green,gotpointercapture@blue,pointerup@blue,click@grey,lostpointercapture@blue', 'An element should only receive click when it is the first common ancestor of pointerdown and pointerup targets'); 
+                            });
+                            phase = PhaseEnum.Phase2;
+                            receivedEvents = [];
+                          } else {
+                            test_pointerEvent.step(function() {
+                              assert_equals(receivedEvents.join(','), 'pointerdown@blue,gotpointercapture@blue,pointerup@blue,click@blue,lostpointercapture@blue', 'An element should only receive click when it is the first common ancestor of pointerdown and pointerup targets'); 
+                            });
+                            test_pointerEvent.done();
+                          }
+                        }
+                        if (event.type == 'pointerdown' && (target == blue || target == green)) {
```

Comment:
I don't have a problem in general testing some of UIEvents here where it's convenient.  Since PE builds in UIEvents it's probably impossible to have tests which DON'T depend on things defined in UIEvents.  But if we find such tests trigger failures in some browsers we care about, we might want to discuss moving them elsewhere (so the failures are accounted for in the right place).